### ISSUE TITLE
sr3 sanity restarts things that are disabled.

### ISSUE THREAD

**Issue by petersilva** [MetPX/sarracenia#647]
to reproduce:

clone insects test suite; prep system to be able to run it.
cd static_flow
./flow_setup.sh config
sr3 disable
sr3 status

```
fractal% sr3 status
status: 
Component/Config                         Processes   Connection        Lag                Rates                                        
                                         State   Run Retry  msg data   LagMax  LagAvg  %rej     pubsub   messages     RxData     TxData 
                                         -----   --- -----  --- ----   ------  ------  ----   --------       ----     ------     ------ 
cpost/pelle_dd1_f04                      disa    1/1          -          -         -     -          -        -
cpost/pelle_dd2_f05                      disa    1/1          -          -         -     -          -        -
cpost/veille_f34                         disa    1/1          -          -         -     -          -        -
cpump/xvan_f14                           disa    1/1          -          -         -     -          -        -
cpump/xvan_f15                           disa    1/1          -          -         -     -          -        -
poll/sftp_f62                            disa    1/1          -          -         -     -          -        -
poll/sftp_f63                            disa    1/1          -          -         -     -          -        -
post/shim_f63                            disa    1/1          -          -         -     -          -        -
post/t_dd1_f00                           disa    1/1          -          -         -     -          -        -
post/t_dd2_f00                           disa    1/1          -          -         -     -          -        -
post/test2_f61                           disa    1/1          -          -         -     -          -        -
report/tsarra_f20                        disa    1/1          -          -         -     -          -        -
sarra/download_f20                       disa    1/1          -          -         -     -          -        -
sender/tsource2send_f50                  disa    1/1          -          -         -     -          -        -
shovel/rabbitmqtt_f22                    disa    1/1          -          -         -     -          -        -
subscribe/amqp_f30                       disa    1/1          -          -         -     -          -        -
subscribe/cdnld_f21                      disa    1/1          -          -         -     -          -        -
subscribe/cfile_f44                      disa    1/1          -          -         -     -          -        -
subscribe/cp_f61                         disa    1/1          -          -         -     -          -        -
subscribe/ftp_f70                        disa    1/1          -          -         -     -          -        -
subscribe/mirror_f80                     disa    1/1          -          -         -     -          -        -
subscribe/q_f71                          disa    5/5          -          -         -     -          -        -
subscribe/rabbitmqtt_f31                 disa    1/1          -          -         -     -          -        -
subscribe/u_sftp_f60                     disa    1/1          -          -         -     -          -        -
watch/f40                                disa    1/1          -          -         -     -          -        -
      Total Running Configs:  25 ( Processes: 0 missing: 0 stray: 0 )
                     Memory: uss:0 Bytes rss:0 Bytes vms:0 Bytes 
                   CPU Time: User:0.00s System:0.00s 
	   Pub/Sub Received: 0 msgs/s (0 Bytes/s), Sent:  0 msgs/s (0 Bytes/s)
	      Data Received: 0 Files/s (0 Bytes/s), Sent: 0 Files/s (0 Bytes/s) 
fractal% 

```

when run sr3 sanity, it should then start some processes... I have not succeeded in reproducing the problem though, it only happens in a different cluster...

**Comment by petersilva** [comment]
Also, disabled state disappears on cleanup....

**Comment by petersilva** [comment]
while working on other issues, noticed that when sr sanity runs, it only runs _find_missing_instances, which generates an initial list of missing instances, but it later gets winnowed out by things like _resolve... so just ran them after the find_missing, and suddenly the list is correct.

It only causes issues when there are missing instances to restart.

**Comment by petersilva** [comment]
fix released with v3.00.37
